plugins/woocommerce-admin/client/wp-admin-scripts/product-tracking/shared.ts

```diff
@@ -324,4 +324,21 @@ export const initProductScreenTracks = () => {
 				} );
 			}
 		} );
+
+	document
+		.querySelector( '#view-product-link' )
+		?.addEventListener( 'click', () => {
+			recordEvent( 'product_view_product_click', getProductData() );
+		} );
+
+	const dismissProductUpdatedButtonSelector =
+		'.notice-success.is-dismissible > button';
```

Comment:
So, not supported well enough by all browsers to actually do, but..

```css
p:has(#view-product-link) ~ .notice-dismiss
```

Would be a bit more robust, since we are in control of the `view-product-link` id (see comment about that name below).

Without `:has()` though, we would have to query for the `#view-product-link` and then use the DOM to get the parent, and then the sibling button.

Not sure that is any more robust. Any solution depends on the DOM for the notice not changing, and we are not in control of that.